app/views/home/_form.html.haml

```diff
@@ -0,0 +1,7 @@
+= simple_form_for :search, url: root_path, method: "GET", html: { class: 'form-inline'} do |f|
+
+  .form-group
+    = f.input :author, placeholder: 'Author', input_html: { value: "" }, label: false
```

Comment:
Is the `input_html: { value: "" }` part necessary?

Reply:
Changed `f.input` to `f.input_field` and `input_html: { value: ' ' }` to `value: ' '`